package.json

```diff
@@ -21,5 +24,8 @@
     "@biomejs/biome": "^1.9.4",
     "@types/node": "^22.10.2",
     "typescript": "^5.7.2"
+  },
+  "dependencies": {
```

Comment:
we don't need that dependency - in fact using a dependency like this is a real risk for typosquatting!  `readline` is a module built into node.js, and even if we needed it (which we don't), wouldn't need to be installed like this - pls remove it